.github/dependabot.yml

```diff
@@ -1,17 +1,27 @@
 version: 2
 updates:
 - package-ecosystem: npm
-  directory: "/"
+  directories:
+    - "/"
+    - "/packages/*"
```

Comment:
this is not a mono-repo, but I just copied the config over from https://github.com/ipfs/helia-verified-fetch/blob/3a299029ca4c276c49b9b8feb6a5e33e8cc8aba0/.github/dependabot.yml